core/tokenRegistry/src/scripts/checkForeignAssetConfig.ts

```diff
@@ -0,0 +1,54 @@
+// patch out annoying logs
+const info = console.info;
+console.info = function (x: any, ...rest: any) {
+  if (x !== 'secp256k1 unavailable, reverting to browser version') {
+    info(x, ...rest);
+  }
+};
```

Comment:
do we have to do this in every file we want it patched for?

Reply:
Not sure 🤔